src/SprykerEco/Zed/Computop/Business/Payment/Reader/ComputopPaymentReader.php

```diff
@@ -35,9 +35,9 @@ public function isComputopPaymentExist(QuoteTransfer $quoteTransfer): QuoteTrans
     {
         /** @var \Generated\Shared\Transfer\ComputopCreditCardPaymentTransfer $paymentTransfer */
         $paymentTransfer = $this->getPaymentTransfer($quoteTransfer);
-        $paymentEntity = $this->queryContainer->queryPaymentByTransactionId($paymentTransfer->getTransId())->findOne();
+        $paymentEntity = $this->queryContainer->queryPaymentByTransactionId($paymentTransfer->getTransIdOrFail())->findOne();
```

Comment:
looks like breaking change
was not required before

Reply:
Fixed

Reply:
Same here, no break if those are required to exist for this to make sense and not run into an error.